guides/source/active_record_callbacks.md

```diff
@@ -289,86 +615,93 @@ The following methods trigger callbacks:
-[`update_counters`]: https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_counters
-[`upsert`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert
-[`upsert_all`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all
+Just as with [validations](active_record_validations.html), it is also possible
+to skip callbacks by using the following methods:
+
+* `decrement!`
+* `decrement_counter`
+* `delete`
+* `delete_all`
+* `delete_by`
+* `increment!`
+* `increment_counter`
+* `insert`
+* `insert!`
+* `insert_all`
+* `insert_all!`
+* `touch_all`
+* `update_column`
+* `update_columns`
+* `update_all`
+* `update_counters`
+* `upsert`
+* `upsert_all`
+
+WARNING. These methods should be used with caution because there may be
+important business rules and application logic in callbacks that you do not want
+to bypass. Bypassing them without understanding the potential implications may
+lead to invalid data.
 
 Halting Execution
 -----------------
 
-As you start registering new callbacks for your models, they will be queued for execution. This queue will include all your model's validations, the registered callbacks, and the database operation to be executed.
+As you start registering new callbacks for your models, they will be queued for
+execution. This queue will include all of your model's validations, the
+registered callbacks, and the database operation to be executed.
 
-The whole callback chain is wrapped in a transaction. If any callback raises an exception, the execution chain gets halted and a ROLLBACK is issued. To intentionally stop a chain use:
+The whole callback chain is wrapped in a transaction. If any callback raises an
+exception, the execution chain gets halted and a **rollback** is issued. To
+intentionally halt a chain use:
 
 ```ruby
 throw :abort
 ```
 
-WARNING. Any exception that is not `ActiveRecord::Rollback` or `ActiveRecord::RecordInvalid` will be re-raised by Rails after the callback chain is halted. Additionally, may break code that does not expect methods like `save` and `update` (which normally try to return `true` or `false`) to raise an exception.
+WARNING. Any exception that is not `ActiveRecord::Rollback` or
+`ActiveRecord::RecordInvalid` will be re-raised by Rails after the callback
+chain is halted. Additionally, it may break code that does not expect methods
+like `save` and `update` (which normally try to return `true` or `false`) to
+raise an exception.
 
-NOTE: If an `ActiveRecord::RecordNotDestroyed` is raised within `after_destroy`, `before_destroy` or `around_destroy` callback, it will not be re-raised and the `destroy` method will return `false`.
+NOTE: If an `ActiveRecord::RecordNotDestroyed` is raised within `after_destroy`,
+`before_destroy` or `around_destroy` callback, it will not be re-raised and the
+`destroy` method will return `false`.
```

Comment:
Maybe something like: 

WARNING: Any exception that is not `ActiveRecord::Rollback` or `ActiveRecord::RecordInvalid` will be re-raised by Rails after the callback chain is halted. This behavior applies not only to operations like save and update, which normally return true or false, but also to destroy operations. If an ActiveRecord::RecordNotDestroyed exception is raised within `after_destroy`, `before_destroy`, or `around_destroy` callbacks, it will not be re-raised, and the `destroy` method will return false.

Wanted your thoughts before incorporating it.